charts/backingservices/charts/srs/README.md

```diff
@@ -61,17 +61,15 @@ To deploy Pega Platform with the SRS backing service, the SRS helm chart require
 | `k8sProvider`                               | Specify your Kubernetes provider name. Supported values are [`eks`, `aks`, `minikube`, `gke`, `openshift`, `pks`].. 
 
 ### Enabling security between SRS and Elasticsearch
-To configure a secure connection between the SRS cluster and Elasticsearch, add the following the settings in your backingservices configuration file to reflect your organization's connectivity setup.
+Enabling a secure connection between SRS and your Elasticsearch cluster depends on the method you chose to deploy the Elasticsearch cluster.
```

Comment:
I think it's better to stay conceptual here and not delve into clusters here, because the requirements are network-based. describing clusters feels like we should be talking about ports and nodes. Just my take, of course. The statement is fine as is...if you use sever, replace cluster with service throughout the readme

Enabling a secure connection between SRS and your Elasticsearch service depends on the method you chose to deploy it.

Reply:
This is a great suggestion, you're right! Thanks!